catalog/pages/breadcrumbs/index.md

```diff
@@ -64,6 +64,7 @@ dark: true
 <ThemeProvider theme={{ themeName: "tm"}}>
   <Breadcrumb>
       <Breadcrumb.Item position="1" role="link" ariaLabel="Home" href="/">Home</Breadcrumb.Item>
+      <Breadcrumb.Item position="1" role="link" ariaLabel="Home" onClick={() => {}}>Path</Breadcrumb.Item>
```

Comment:
Do you want to leave this?

Reply:
Yep, I missed these styles because the catalog didn't include this use case.